node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -375,6 +435,11 @@ deriveSafeCopySimple 0 'base ''CTxMeta
 deriveSafeCopySimple 0 'base ''CUpdateInfo
 deriveSafeCopySimple 0 'base ''AddressLookupMode
 deriveSafeCopySimple 0 'base ''CustomAddressType
+deriveSafeCopySimple 0 'base ''TxAux
```

Comment:
Why is it here and not in `Pos.SafeCopy`? I mean `TxAux`, I dunno about other types, I guess they should be here, but for `TxAux` it seems more reasonable to put it into `Pos.SafeCopy`. Or move `Pos.SafeCopy` somewhere into wallet namespace.

Reply:
Thanks, I'll fix this along with changes requested by @pva701